content/docs/how-tos/onboarding-a-new-component.md

```diff
@@ -0,0 +1,295 @@
+
+{{< highlight bash >}}
+make jobs
+{{< / highlight >}}
+
+This will create all necessary files under `ci-operator/jobs/$org/$repo`, creating a good default set of Prow jobs.
+
+#### Setting up team ownership of ci-operator and Prow config files
+
+While the initial PR to [`openshift/release`](https://github.com/openshift/release) will need to be reviewed and
+approved by [root approvers](https://github.com/openshift/release/blob/master/OWNERS), once the component config is in
+place, it should be owned by the component team. To achieve this, an `OWNERS` file mirroring what exists upstream should
+be placed into both `ci-operator/config/$org/$repo` and `ci-operator/jobs/$org/$repo` directories.
+
+### Enabling Automatic Merges
+
+Prow’s `tide` component periodically searches for pull requests that fit merge criteria (for instance, presence of a `lgtm`
+label and absence of the `do-not-merge/hold` label) and merges them. `Tide` furthermore requires not only that all required
+tests in the Prow configuration succeed and all posted statuses on the GitHub pull request are green but also that the
+tests tested the latest commit in the pull request on top of the latest commit in the branch that the pull request is
+targeting before a pull request is considered for merging.
+
+If you want to configure automatic merges for your component repository, you can either write a new merge criterion for
+the repository or add your repository to one of the existing configurations. Tide’s configuration is documented
+[upstream](https://github.com/kubernetes/test-infra/blob/master/prow/cmd/tide/README.md). The existing Tide
+configuration for our CI system lives in
+[`config.yaml`](https://github.com/openshift/release/blob/master/core-services/prow/02_config/_config.yaml) in the
+[openshift/release](https://github.com/openshift/release) repository under the `config.yaml["tide"]["queries"]` key. If
+your repository does not have `OWNERS` files, or if you have not chosen to opt into the `/approve `process, it is
+suggested that you add your repository to the merge criterion that requires only `/lgtm`.
+
+#### Who can `/approve`?
+
+Repo’s `OWNERS` and `OWNERS_ALIASES` define the list. It is a concept of Prow workflow. Those files also define who
+could get selected as reviewers of PRs in that repo. See [Prow's
+doc](https://github.com/kubernetes/test-infra/blob/master/prow/plugins/approve/approvers/README.md#approval-handler-and-the-approved-label)
+on this topic.
+
+#### Who can `/lgtm`?
+
+Github’s users who are the repo’s collaborators. It is a [concept of
+github.](https://developer.github.com/v3/projects/collaborators/) Contributors should follow this
+[mojo](https://mojo.redhat.com/docs/DOC-1081313#jive_content_id_Github_Access) to become a collaborator for repositories
+in openshift org. See [Prow’s
+doc](https://github.com/kubernetes/test-infra/blob/master/prow/plugins/approve/approvers/README.md#lgtm-label) on this
+topic.
+
+## CI Operator Configuration
```

Comment:
We should see how much of this section we really need, it's mostly duplicate of the existing architecture page, right?